plugins/woocommerce/src/Internal/Admin/Notes/WooSubscriptionsNotes.php

```diff
@@ -75,9 +75,16 @@ public function update_option_woocommerce_helper_data( $old_value, $value ) {
 	}
 
 	/**
-	 * Things to do on admin_init.
+	 * Things to do on admin_head.
 	 */
-	public function admin_init() {
+	public function admin_head() {
+		if ( ! \Automattic\WooCommerce\Admin\PageController::is_admin_or_embed_page() ) {
```

Comment:
How about adding `use \Automattic\WooCommerce\Admin\PageController` and replacing this line with 
```suggestion
		if ( ! PageController::is_admin_or_embed_page() ) {
```